sql/mysqld.cc

```diff
@@ -9495,7 +9495,7 @@ extern "C" void *start_wsrep_THD(void *arg) {
   thd_manager->add_thd(thd);
   thd_added = true;
 
-#ifdef SKIP_INNODB_HP
+#ifndef SKIP_INNODB_HP
```

Comment:
The commit message only mentions test fixes. Please also mention the code changes in the commit message.

Reply:
fixed